runner/src/server/plugins/engine/models/submission/NotifyModel.ts

```diff
@@ -38,7 +28,7 @@ export function NotifyModel(
   model: FormModel,
   outputConfiguration: NotifyOutputConfiguration,
   state: FormSubmissionState
-): NotifyModel {
+): TNotifyModel {
```

Comment:
I was getting clashes with `NotifyModel` the function and `NotifyModel` the data type, so I've renamed it to `TNotifyModel`